src/Illuminate/Support/Str.php

```diff
@@ -271,12 +271,11 @@ public static function randomBytes($length = 16)
      * Should not be considered sufficient for cryptography, etc.
      *
      * @param  int  $length
+     * @param  string  $pool
      * @return string
      */
-    public static function quickRandom($length = 16)
+    public static function quickRandom($length = 16, $pool = '0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ')
```

Comment:
What i meant was just a standard ternary operation as is common with all nullable default implementations.